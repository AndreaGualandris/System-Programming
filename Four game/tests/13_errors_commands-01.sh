#!/bin/bash
exec "$1" 7 5 o <<EOF
2 x y z 3 4 ciaomamma 5 q
EOF
