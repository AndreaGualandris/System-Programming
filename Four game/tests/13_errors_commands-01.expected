-------
       
       
       
       
       
-------
0123456
Player 'o' to move:
  o
  v
-------
       
       
       
       
  o    
-------
0123456
Player 'x' to move:
Unknown command.
Unknown command.
Unknown command.
   x
   v
-------
       
       
       
       
  ox   
-------
0123456
Player 'o' to move:
    o
    v
-------
       
       
       
       
  oxo  
-------
0123456
Player 'x' to move:
Unknown command.
Unknown command.
Unknown command.
Unknown command.
Unknown command.
Unknown command.
Unknown command.
Unknown command.
Unknown command.
     x
     v
-------
       
       
       
       
  oxox 
-------
0123456
Player 'o' to move:
